src/base.jl

```diff
@@ -291,15 +339,16 @@ function IS.add_forecast!(
 
     if component isa LoadZones
         uuids = Set([IS.get_uuid(x) for x in get_buses(component)])
-        for component_ in (load for load in IS.get_components(ElectricLoad, data)
-                          if get_bus(load) |> IS.get_uuid in uuids)
+        for _component in (load for load in IS.get_components(ElectricLoad, data)
+                          if IS.get_uuid(get_bus(load)) in uuids)
             if forecast isa IS.DeterministicInternal
-                forecast_ = IS.DeterministicInternal(IS.get_label(forecast), ts_data)
+                _forecast = IS.DeterministicInternal(IS.get_label(forecast), ts_data)
             else
                 # TODO: others
                 error("forecast type is not supported yet: $(typeof(forecast))")
             end
-            IS.add_forecast!(data, component_, forecast, ts_data)
+            # FIXME: Should this be `_forecast` or `forecast`?
```

Comment:
Poorly written code that will do the same thing either way.  I'll fix this separately.

Reply:
Will be resolved in a separate PR.